VideoLocker/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -91,6 +96,11 @@ public void onActivityCreated(Bundle savedInstanceState) {
             logger.error(ex);
         }
         loadData(getView());
+
+        String lastAccessedId = getArguments().getString(Router.EXTRA_LAST_ACCESSED_ID);
```

Comment:
hmm ... nice catch and in that way we'll also get rid of that unnecessary generic exception-catch block :+1: